boardwalk/dev.yml

```diff
@@ -100,7 +103,6 @@ services:
     build: boardwalk
     environment:
       - BOARDWALK_HOST=${dcc_dashboard_host}
-      - BW_DATA_URL=${dcc_dashboard_protocol}://${dcc_dashboard_host}
```

Comment:
This should not be showing up as change. Can you please resolve it so it doesn't.

Reply:
We decided not to worry about this as the branch merges with `feature/commons`.